lib/ci.js

```diff
@@ -22,7 +23,8 @@ const CI_TYPES = new Map([
   [LIBUV, { name: 'libuv', re: /libuv/ }],
   [NOINTL, { name: 'No Intl', re: /nointl/ }],
   [V8, { name: 'V8', re: /node-test-commit-v8/ }],
-  [LINTER, { name: 'Linter', re: /node-test-linter/ }]
+  [LINTER, { name: 'Linter', re: /node-test-linter/ }],
+  [LITE, { name: 'Lite', re: /node-test-commit-lite/ }]
```

Comment:
It is node-test-pull-request-lite and node-test-commit-light

Reply:
@gibfahn renamed `node-test-commit-light` to `node-test-commit-lite`: https://github.com/nodejs/build/issues/1178

Reply:
Oh, missed that one. But the PR job should still be included

Reply:
https://github.com/nodejs/node-core-utils/pull/171#discussion_r167386568

Reply:
@richardlau It doesn’t seem right to capture that as a Full CI when we have a new category named Lite?

Reply:
In that case the regexp for full would also need to be modified to stop it matching.